Mathlib/RepresentationTheory/Homological/GroupHomology/LowDegree.lean

```diff
@@ -625,4 +646,133 @@ theorem isBoundary₂_of_mem_boundaries₂
     IsBoundary₂ x := hx
 
 end ofDistribMulAction
+
+open ShortComplex
+
+section cyclesIso₀
+
+instance : Epi (shortComplexH0 A).g := inferInstanceAs <| Epi ((coinvariantsMk k G).app A)
+
+lemma shortComplexH0_exact : (shortComplexH0 A).Exact := by
+  rw [ShortComplex.moduleCat_exact_iff]
+  intro x (hx : Coinvariants.mk _ _ = 0)
+  rw [Coinvariants.mk_eq_zero, ← range_d₁₀_eq_coinvariantsKer] at hx
+  rcases hx with ⟨x, hx, rfl⟩
+  use x
+  rfl
+
+variable [DecidableEq G]
+
+/-- The 0-cycles of the complex of inhomogeneous chains of `A` are isomorphic to `A`. -/
+def cyclesIso₀ : cycles A 0 ≅ A.V :=
+  (inhomogeneousChains A).iCyclesIso _ 0 (by aesop) (by aesop) ≪≫ chainsIso₀ A
+
+@[reassoc (attr := simp), elementwise (attr := simp)]
+lemma cyclesIso₀_inv_comp_iCycles :
+    (cyclesIso₀ A).inv ≫ iCycles A 0 = (chainsIso₀ A).inv := by
+  simp [cyclesIso₀]
+
+/-- The arrow `(G →₀ A) --d₁₀--> A` is isomorphic to the differential
+`(inhomogeneousChains A).d 1 0` of the complex of inhomogeneous chains of `A`. -/
+@[simps! hom_left hom_right inv_left inv_right]
+def d₁₀ArrowIso :
+    Arrow.mk ((inhomogeneousChains A).d 1 0) ≅ Arrow.mk (d₁₀ A) :=
+  Arrow.isoMk (chainsIso₁ A) (chainsIso₀ A) (comp_d₁₀_eq A)
+
+/-- The 0-cycles of the complex of inhomogeneous chains of `A` are isomorphic to
+`A.ρ.coinvariants`, which is a simpler type. -/
+def opcyclesIso₀ : (inhomogeneousChains A).opcycles 0 ≅ (coinvariantsFunctor k G).obj A :=
+  CokernelCofork.mapIsoOfIsColimit
+    ((inhomogeneousChains A).opcyclesIsCokernel 1 0 (by simp)) (shortComplexH0_exact A).gIsCokernel
+      (d₁₀ArrowIso A)
+
+@[reassoc (attr := simp), elementwise (attr := simp)]
+lemma pOpcycles_comp_opcyclesIso_hom :
+    (inhomogeneousChains A).pOpcycles 0 ≫ (opcyclesIso₀ A).hom =
+      (chainsIso₀ A).hom ≫ (coinvariantsMk k G).app A :=
+  CokernelCofork.π_mapOfIsColimit (φ := (d₁₀ArrowIso A).hom) _ _
+
+@[reassoc (attr := simp), elementwise (attr := simp)]
+lemma coinvariantsMk_comp_opcyclesIso₀_inv :
+    (coinvariantsMk k G).app A ≫ (opcyclesIso₀ A).inv =
+      (chainsIso₀ A).inv ≫ (inhomogeneousChains A).pOpcycles 0 :=
+  (CommSq.vert_inv ⟨pOpcycles_comp_opcyclesIso_hom A⟩).w
+
+end cyclesIso₀
+
+section isoCycles₁
+
+variable [DecidableEq G]
+
+/-- The short complex `(G² →₀ A) --d₂₁--> (G →₀ A) --d₁₀--> A` is isomorphic to the 1st
+short complex associated to the complex of inhomogeneous chains of `A`. -/
+@[simps! hom inv]
+def shortComplexH1Iso : (inhomogeneousChains A).sc 1 ≅ shortComplexH1 A :=
```

Comment:
This name suggests flipping the two sides of the iso... or flipping the name...

Reply:
flipped the name, thanks!